contracts/interfaces/IQuest.sol

```diff
@@ -6,6 +6,8 @@ pragma solidity =0.8.16;
 interface IQuest {
     // This event is triggered whenever a call to #claim succeeds.
     event Claimed(address account, address rewardAddress, uint256 amount);
+    event Started(uint timestamp);
```

Comment:
ugh this is still confusing. Wondering if we rename this to queue

Reply:
I'm cool with it, but should we rename just the event or the function as well?